dmsnitch/resource_snitch.go

```diff
@@ -166,6 +188,7 @@ func resourceSnitchRead(d *schema.ResourceData, m interface{}) error {
 		d.Set("status", snitch.Status)
 		d.Set("interval", snitch.Interval)
 		d.Set("type", snitch.Type)
+		d.Set("tags", emailList)
```

Comment:
```suggestion
		d.Set("alert_emails", emailList)
```
This looks like a copy/paste error